src/common/files_js.ml

```diff
@@ -13,10 +13,9 @@
 open Utils
 open Modes_js
 
-let flow_extensions = [
-    ".js"  ;      (* Standard JavaScript files *)
-    ".jsx" ;      (* JavaScript files with JSX *)
-  ]
+let get_flow_extensions root =
```

Comment:
It's a shame that we have to thread `root` through these just for purposes of some internal implementation logic.

It seems `files_js.ml` has an `init` method already, though, for initializing stuff like storing the `config`.Can we just pass the `options` object in through there and store it at the toplevel?

(similar to how things work over in [`module_js.ml`](https://github.com/facebook/flow/blob/master/src/typing/module_js.ml#L472) )